### ISSUE TITLE
About the choices in LLaVA+S^2 implementation

### ISSUE THREAD

**Issue by jungle-gym-ac** [bfshi/scaling_on_scales#10]
Great work! I've read the paper and it seems the `LLaVA+S^2` is implemented with OpenCLIP VIsion Encoder, and the LLM is finetuned with LoRA. However, the LLaVA baseline you compared with is implemented with OpenAI-CLIP Vision Encoder, and the LLM is full-finetuned(without LoRA).

If I'm right, I just wonder if you have tried using the same Vision Encoder, or full-finetuning the LLM, and what are the results of this setting? Thank you.

**Comment by bfshi** [comment]
Hi @jungle-gym-ac, yeah good question. In the scaling experiment on llava (Fig 3 in the paper), all the models including the baselines use openclip. The experiment of comparing llava-s2 to official llava (Table 11 in Appendix) uses OpenAI clip.

And you are right, all the models we trained on llava use lora while the official llava checkpoint we compare to uses full fine tuning. According to the official llava repo, the performance of llava with ft/lora doesn't differ much on average, but yeah comparing to the official checkpoint Lora would be fairer. We will include this in a later version of the paper. And we didn't try llava-s2 with ft.

**Comment by zui-jiang** [comment]
@bfshi I'm wondering is the pretraining data used in LLaVA+S^2 the same as the original LLaVA? cuz I tried pretraining LLaVA with S^2, and the loss is much bitter than LLaVA (2.5 compared with 0.7)

**Comment by bfshi** [comment]
The training recipe is the exact same one as LLaVA. Loss of 2.5 seems
weird. That is almost the same loss as an untrained model. Did you change
anything from the llava repo that may cause some unexpected behaviors?
